### PR TITLE
Adds Lord Skib to the game.

### PR DESCRIPTION
**Pull request by CBozo**
Dop dop? Dop dop yes... Hmmmmmm... Skibidi...

## Summary by CodeRabbit

- **New Features**
	- Introduced "Lord Skib," a new character in Goobstation with unique abilities and visual representation.
	- Added interactive features including announcements, movement mechanics, and communication capabilities.
	- Enhanced gameplay dynamics with components for physics, zooming, and event interactions.

### CODE REVIEW

File: Resources/Prototypes/Goobstation/Entities/Mobs/Player/LordSkib.yml [hunk 1]
```diff
@@ -0,0 +1,80 @@
+- type: entity
```
Comment (whateverusername0) [inline Resources/Prototypes/Goobstation/Entities/Mobs/Player/LordSkib.yml:1]: delete this

File: Resources/Prototypes/Goobstation/Entities/Mobs/Player/LordSkib.yml [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+  components:
+  - type: Sprite
+    sprite: Goobstation/Mobs/Demons/skibidi.rsi
+    drawdepth: Ghosts
+    scale: 10,10
+    layers:
+    - state: alive
+      shader: unshaded
+
+- type: entity
```
Comment (whateverusername0) [inline Resources/Prototypes/Goobstation/Entities/Mobs/Player/LordSkib.yml:15]: and this

**Review by whateverusername0 — COMMENTED** [review]
god

**Comment by CBozo** [comment]
what the fuck is a localization

**Pull request closed without merging**